### ISSUE TITLE
Multiple elements on the static site with the same id

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4039]
## Area or Component
Static site, all support/guidance pages

## Quick Summary
There are two `<div>` elements at the bottom of the page with the same id. One is empty and appears to have no functionality.

### How to Replicate
Visit https://stg.simplereport.gov/getting-started/ and search for `usasearch_sayt` in the inspector tab

## Accessibility Violation
id attribute value must be unique: https://dequeuniversity.com/rules/axe/4.4/duplicate-id?application=AxeChrome

## Action Requested
Remove the duplicate empty div

## Original Deque Issue
https://docs.google.com/spreadsheets/d/16udNpwdJ8sltAULnmSIGgkWVQsfuFvcoymP0-s_QwfE/edit#gid=1655325714&range=4:4

**Comment by emmastephenson** [comment]
I can't replicate this issue in production anymore, so I'm guessing it was resolved with one of our other static site tickets 🤷
